test/test_custom.py

```diff
@@ -85,18 +94,39 @@ def test_round_trip(self):
         obj.foo_bool.append(True)
         obj.foo_bool.append(False)
         self.assertEqual([True, False], obj.foo_bool)
+        obj_unregistered_name = 'bool_test_unregistered'
+        obj_u = CustomUnregisteredTopClass(obj_unregistered_name)
+        obj_u.foo_bool.append(True)
+        self.assertEqual([True], obj_u.foo_bool)
         doc = sbol3.Document()
         doc.add(obj)
+        doc.add(obj_u)
         doc2 = sbol3.Document()
         # Round trip the document
         with tempfile.TemporaryDirectory() as tmpdirname:
             test_file = os.path.join(tmpdirname, 'custom.nt')
             doc.write(test_file, sbol3.NTRIPLES)
             doc2.read(test_file, sbol3.NTRIPLES)
         obj2 = doc2.find(obj_name)
+        obj_u2 = doc2.find(obj_unregistered_name)
         # The lists are necessarily unordered because of RDF
-        # Compare specially
-        self.assertCountEqual([True, False], obj2.foo_bool)
+        self.assertEqual([False, True], sorted(obj2.foo_bool))
+        unregistered_values = obj_u2._properties['https://github.com/synbiodex/pysbol3#customBool']
+        self.assertEqual(['true'], [str(x) for x in unregistered_values])
+
+        # Finally, make sure the objects can be copied into a new document
+        doc3 = sbol3.Document()
+        # This test deliberately uses the old copy function, as that was where an error previously occurred
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", DeprecationWarning)
+            obj2.copy(doc3)
+            obj_u2.copy(doc3)
+        obj3 = doc3.find(obj_name)
+        obj_u3 = doc3.find(obj_unregistered_name)
+        # The lists are necessarily unordered because of RDF
+        self.assertEqual([False, True], sorted(obj3.foo_bool))
```

Comment:
Consider using [assertCountEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual) here instead of sorting the list.